packages/ui/src/components/va-data-table/VaDataTable.vue

```diff
@@ -124,7 +124,10 @@
             v-for="row in rows"
             :key="`table-row_${row.initialIndex}`"
             class="va-data-table__table-tr"
-            :class="{ selected: isRowSelected(row) }"
+            :class="[
+              isRowSelected(row) ? 'selected' : '',
```

Comment:
```suggestion
              { 'selected':  isRowSelected(row) },
```

Reply:
Ah, didn't notice, you changed that contruction. Anyway, it also nicely works as a part of classes array.